CMakeLists.txt

```diff
@@ -180,7 +180,7 @@ if (ONEDPL_BACKEND MATCHES "^(tbb|dpcpp|dpcpp_only)$")
 
     target_compile_definitions(oneDPL INTERFACE
         $<$<CONFIG:Debug>:TBB_USE_DEBUG=1>
-        $<$<CONFIG:Debug>:PSTL_USE_DEBUG>
+        $<$<CONFIG:Debug>:PSTL_USE_DEBUG=1>
```

Comment:
`$<$<CONFIG:Debug>:PSTL_USE_DEBUG=1>` should not depend on a backend type. Let's move it outside of the backend-specific branches to apply everywhere.

Reply:
I agree and have moved it outside of backend specific branches.